components/ReceiptPreview.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { capitalize } from '../utils';
+
+function Card(props) {
+  const { header, donation } = props.data;
+  const { donor } = donation;
+  const full_name = `${donor.name} ${donor.middlename} ${donor.surname}`;
+
+  return (
+    <div className="card shadow-xl p-4 mt-5">
+      <div className="grid-container grid grid-cols-8">
+        <div className="item1 col-span-3">
+          <div className="-mt-1">
```

Comment:
why the `-` sign at the beginning? What does it do?

Reply:
`-mt-1` is Tailwind class for negative margin top